Core/clim-core/panes.lisp

```diff
@@ -363,11 +363,38 @@ order to produce a double-click")
   (print-unreadable-object (pane sink :type t :identity t)
     (prin1 (pane-name pane) sink)))
 
+(defgeneric find-concrete-pane-class (pane-realizer pane-type)
+  (:documentation "Resolves abstract pane name to a concrete pane ~
+class which may be instatinated. Function specializes on its first ~
+argument. Returns NIL if no concrete class exists.")
```

Comment:
* There are `~` characters at the end of two lines

* I would write the first sentence as "Resolve ABSTRACT-PANE-TYPE to a concrete pane class", but I guess that is another question for the coding style document.

* > Function specializes on its first argument.

  I'm not sure what this means for a generic function.

* The default method can also signal an error but the documentation string sounds as if that is not a possibility. One should change.

Reply:
fixed